release notes/v0.45.0.md

```diff
@@ -0,0 +1,169 @@
+k6 v0.45.0 is here 🎉! This release includes:
+
+- Experimental gRPC streaming support.
+- Update scripts in the cloud without running tests.
+- JS Metadata API.
+- A lot of internal changes and bugfixes.
+
+## Breaking changes
+
+* [#3066](https://github.com/grafana/k6/pull/3066) k6 will warn users about metric names incompatible with OpenTelemetry or Prometheus. This is planned to become an error with v0.48.0. 
```

Comment:
We should also update our docs with the allowed pattern and link here. (duplicating the comment since it was the part of the previous suggestion)

Reply:
The issue is already marked (by you) with a documentation needed. Do you want us to expand on this in the comment?

Reply:
I'd prefer to keep the comment till the actual moment when we refer here to the docs, otherwise, it could be missed.